auth0_flutter/FAQ.md

```diff
@@ -116,6 +117,33 @@ This library has no control whatsoever over the alert box. Its contents cannot b
 
 This library has no control whatsoever over the alert box. It cannot be closed programmatically. Unfortunately, that's a limitation of `ASWebAuthenticationSession`.
 
+### 6. How to resolve the _Failed to start this transaction, as there is an active transaction at the moment_ error?
+
+Users might encounter this error when the app moves to the background and then back to the foreground while the login/logout alert box is displayed, for example by locking and unlocking the device. The alert box would get dismissed but when the user tries to log in again, the Web Auth operation fails with the `transactionActiveAlready` error.
+
+This is a known issue with `ASWebAuthenticationSession` and it is not specific to auth0-flutter. We have already filed a bug report with Apple and are awaiting for a response from them.
+
+#### Workarounds
+
+##### Clear the login transaction when handling the `transactionActiveAlready` error
+
+You can invoke `cancelWebAuth()` to manually clear the current login transaction upon encountering this error. Then, you can retry login. For example:
+
+```dart
+try {
+    await webAuth.login(useHTTPS: false);
+} catch (e) {
+    if (e.toString() ==
+    'UNKNOWN: Failed to start this transaction, as there is an active transaction at the moment.') {
```

Comment:
Let's avoid relying on error descriptions as much as possible, as they are not part of the public API.

In this case, a new error code could be added here: https://github.com/auth0/auth0-flutter/blob/main/auth0_flutter/darwin/Classes/WebAuth/WebAuthExtensions.swift#L12-L20

So the comparison suggested here relies only on the error code.